Neos.Fusion/Classes/Core/ObjectTreeParser/FilePatternResolver.php

```diff
@@ -56,13 +57,14 @@ class FilePatternResolver
      * @return array|string[]
      * @throws Fusion\Exception
      */
-    public static function resolveFilesByPattern(string $filePattern, ?string $filePathForRelativeResolves = null, string $defaultFileEndForUnspecificGlobbing = '.fusion'): array
+    public static function resolveFilesByPattern(string $filePattern, ?string $filePathForRelativeResolves, string $defaultFileEndForUnspecificGlobbing): array
```

Comment:
Why do you remove the defaults here? This would be a breaking change.

Reply:
No its internal (was a protected method before Neos 8) ;) And my previous ME liked defaults, i hate em now ^^